apps/web/frontend/components/weekSelector.tsx

```diff
@@ -1,85 +1,69 @@
-import {
-  eachWeekOfInterval,
-  endOfWeek,
-  endOfYear,
-  format,
-  getWeek,
-  getYear,
-  isSameYear,
-  isThisWeek,
-  setWeek,
-  setYear,
-  startOfWeek,
-  startOfYear,
-} from 'date-fns'
+import { endOfWeek, format, getWeek, getYear, isThisWeek, startOfWeek, nextMonday, previousMonday } from 'date-fns'
+import { enGB } from 'date-fns/locale'
```

Comment:
Love the idea of using locale here.

Can you try to set the locale globally in `_app.page.tsx`: https://date-fns.org/v2.29.3/docs/setDefaultOptions